src/idl_gen_java.cpp

```diff
@@ -821,9 +855,9 @@ class JavaGenerator : public BaseGenerator {
           for (auto kit = fields.begin(); kit != fields.end(); ++kit) {
             auto &key_field = **kit;
             if (key_field.key) {
-              auto qualified_name = WrapInNameSpace(sd);
+              auto qualified_name = namer_.NamespacedType(sd);
```

Comment:
I think this is wrong, since it is doing Uppercamel for namespaces on a type, which should be lower case?